library/src/main/java/com/pokegoapi/api/pokemon/Pokemon.java

```diff
@@ -170,9 +172,12 @@ public Result transferPokemon() throws LoginFailedException, RemoteServerExcepti
 	 * @return the boolean
 	 * @throws LoginFailedException  the login failed exception
 	 * @throws RemoteServerException the remote server exception
+	 * @throws InvalidCurrencyException the currency has exception
 	 */
-	public boolean canPowerUp() throws LoginFailedException, RemoteServerException {
-		return getCandy() >= getCandyCostsForPowerup();
+	public boolean canPowerUp() throws LoginFailedException, RemoteServerException, InvalidCurrencyException {
+		return getCandy() >= getCandyCostsForPowerup()
```

Comment:
Take out the exception and compile it. Why do you think both PRs handled that exception in one way or another. If you feel there shouldn't be an exception, then fix the original class to stop passing it to the reference. That isn't within the scope of this PR.